src/common/xo/index.js

```diff
@@ -711,15 +713,16 @@ export const disableSchedule = id => (
   )
 )
 
-export const deleteSchedule = async schedule => {
+export const deleteBackupSchedule = async schedule => {
   await confirm({
-    title: _('deleteJob'),
-    body: _('deleteJobQuestion')
+    title: _('deleteBackupJob'),
```

Comment:
Is it normal that the method is called `deleteBackupSchedule` and the message id is called `deleteBackupJob`?